airbyte-integrations/bases/debezium-v1-9-6/build.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
 
     implementation 'io.debezium:debezium-api:1.9.6.Final'
     implementation 'io.debezium:debezium-embedded:1.9.6.Final'
-//    implementation 'io.debezium:debezium-connector-sqlserver:1.9.2.Final'
+    implementation 'io.debezium:debezium-connector-sqlserver:1.9.6.Final'
```

Comment:
I just wanted to make sure that what was forgotten in Postgres (`TINYINT(1)`?) is taken care of here.